src/transaction/payment.ts

```diff
@@ -30,11 +30,22 @@ import {Amount, Adjustment, MaxAdjustment,
   limitQuality?: boolean
 }
 
+function isMaxAdjustment(
+  source: Adjustment | MaxAdjustment): source is MaxAdjustment {
```

Comment:
Nitpick: why is this on a new line?

Reply:
exceeds 80 character max, enforced by our linting config :/

Reply:
Ah, I see. Looks strange to me cause I'm used to the "standard" style